paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -138,19 +142,8 @@ internal class PaymentOptionsViewModel(
         }
 
         @Inject
-        lateinit var eventReporter: EventReporter
```

Comment:
When using a `subcomponent`, it will inherit all objects created in the base `component`, so these are all provided by `FlowControllerComponent`, specifically, the `eventReporter` is provided in [PaymentSheetCommonModule](https://github.com/stripe/stripe-android/blob/52eb88f8d0504f5d0429a2431b864b0a4f56ef39/paymentsheet/src/main/java/com/stripe/android/paymentsheet/injection/PaymentSheetCommonModule.kt#L29) by `@Binds`

Reply:
slowly starting to understand dagger haha. Thanks!